.changeset/four-lobsters-search.md

```diff
@@ -0,0 +1,4 @@
+---
+'@finos/legend-server-depot': patch
```

Comment:
I think we need to document this as a `major` and document about the breaking change